src/components/dialog/dialog.spec.js

```diff
@@ -168,6 +168,27 @@ describe('$mdDialog', function() {
       expect(content.id).toBe('dialogContent_' + dialog[0].id);
     }));
 
+    it('should not clobber the id from `md-dialog` when there is no content', inject(function ($mdDialog, $rootScope, $document) {
```

Comment:
Gotcha; I didn't catch that the test didn't have the `<md-dialog-content>`. Thanks for clarifying :wink: